### PR TITLE
Add stEvmos Perpetuals Pool 

### PR DESCRIPTION
**Pull request by LPX55**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Github org/user (Optional, if your code is open source, we can track activity):

### CODE REVIEW

File: projects/unidex/index.js [hunk 1]
```diff
@@ -73,6 +73,15 @@ async function BaseTvl(_time, _ethBlock, { base: block }) {
   const owners = Object.values(contracts)
   return sumTokens2({ chain, block, tokens, owners, })
 }
+async function EvmosTvl(_time, _ethBlock, { evmos: block }) {
+  const contracts = {
+    "stevmosPool": "0x21708707f03A19C3a4ea5E1a132B5cF96b86F294",
+  };
+  const chain = 'evmos'
+  const tokens = [nullAddress]
```
Comment (g1nt0ki) [inline projects/unidex/index.js:81]: also, stvEvmos token? `0x21708707f03a19c3a4ea5e1a132b5cf96b86f294

**Comment by llamatester** [comment]
The adapter at projects/unidex exports TVL: 
        
 
 
    optimism                  39.77 k
    base                      119.24840516915313
    boba                      4.70 k
    arbitrum                  2.54 k
    era                       97.02 k
    metis                     92.91 k
    bsc                       4.30 k
    fantom                    263.01 k
    evmos                     26.05 k
    
    total                    530.40 k

**Comment by LPX55** [comment]
> The adapter at projects/unidex exports TVL:
> 
> ```
> optimism                  39.77 k
> base                      119.24840516915313
> boba                      4.70 k
> arbitrum                  2.54 k
> era                       97.02 k
> metis                     92.91 k
> bsc                       4.30 k
> fantom                    263.01 k
> evmos                     26.05 k
> 
> total                    530.40 k 
> ```

any tips on how to fix the stEVMOS pricing? 

Assuming it's using the API as the source. Both CoinGecko and [Llama server](https://coins.llama.fi/prices/current/evmos:0x2c68d1d6ab986ff4640b51e1f14c716a076e44c4) return an incorrect value. [The DEX API](https://api.geckoterminal.com/api/v2/simple/networks/evmos/token_price/0x2c68d1d6ab986ff4640b51e1f14c716a076e44c4) on the other hand gets the correct value.

**Comment by g1nt0ki** [comment]
@LPX55 I think we are pulling the price from some dex (dunno which) for this, not from coingecko (they dont have price) and we cant pull from coingecko terminal

can you share some dex pool with enough LP? We can price it from there.

Merging this PR for now, thanks.

**Pull request merged**